src/routes/Wallets/routes/Backup/WalletsBackupView.js

```diff
@@ -40,21 +44,43 @@ class WalletsBackupView extends Component<Props> {
     this.props.closeView()
   }
 
+  getData = (isMnemonic: boolean) => {
```

Comment:
for future, better to avoid such generic names
something
`createPrivateKeyRange`
`privateKeyString`
would be better

Reply:
they mean nothing, hard to remember impossible to jump to with IDE